app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -202,4 +203,115 @@ class PrescriptionRepositoryAndroidTest {
     )
     assertThat(database.prescriptionDao().getOne(prescribedDrug.uuid)).isEqualTo(expectedPrescribedDrug)
   }
+
+  @Test
+  fun updating_prescribed_drug_teleconsultation_id_should_work_correctly() {
+    // given
+    val durationToAdvanceBy = Duration.ofMinutes(10)
+    val teleconsultationId = UUID.fromString("35756e86-4b31-44cb-bd5f-cd3bc8ad2448")
+    val timestamps = Timestamps(
+        createdAt = Instant.now(clock),
+        updatedAt = Instant.now(clock),
+        deletedAt = null,
+    )
```

Comment:
Just FYI, we already have a `Timestamps.create(UtcClock)` factory function which does the same thing.